package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "adapt-navigationTitle",
+  "version": "1.0.0",
+  "framework": ">=5.20.0",
```

Comment:
With the removal of `.u-float-left` from the less this should be bumped to 5.24.4